### DISCUSSION TITLE
Academic Institute Banasthali Vidyapeeth Verification

### DISCUSSION THREAD

**Discussion by suryabioinfo** [RosettaCommons/rosetta#100] [ROSIE] [answered]
Dear support team
Banasthali Vidyapeeth is, a University dedicated to Women only education, loacated in Rajasthan Province of India.  We are small group of computational biologists hosted in the department of Bioscience and Biotechnology.
Below are credentials for my academic organization could you please add it to verified list. 
http://www.banasthali.org/
suryapratapsingh@banasthali.in
Thank you in anticipation

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know, - this should be fixed now. Please feel free to re-open if you still having issues.
